mqtt.kafka.broker/README.md

```diff
@@ -1,215 +1,13 @@
-+ kubectl exec --namespace zilla-mqtt-kafka-broker pod/kafka-74675fbb8-w42xt -- /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server localhost:9092 --create --topic mqtt-sessions --config cleanup.policy=compact --if-not-exists
-Created topic mqtt-sessions.
-+ kubectl port-forward --namespace zilla-mqtt-kafka-broker service/zilla-mqtt-kafka-broker 7183 7883
-+ nc -z localhost 7183
-+ kubectl port-forward --namespace zilla-mqtt-kafka-broker service/kafka 9092 29092
-+ sleep 1
-+ nc -z localhost 7183
-Connection to localhost port 7183 [tcp/ibm-mqisdp] succeeded!
-+ nc -z localhost 9092
-Connection to localhost port 9092 [tcp/XmlIpcRegSvc] succeeded!
-```
-
-### Install mqtt client
-
-Requires MQTT 5.0 client, such as Mosquitto clients.
-
-```bash
-brew install mosquitto
-```
-
-### Install kcat client
-
-Requires Kafka client, such as `kcat`.
-
-```bash
-brew install kcat
-```
-
-### Verify behavior
-
-Connect two subscribing clients first, then send `Hello, world` from publishing client. Verify that the message arrived to Kafka.
-
-```bash
-mosquitto_sub -V '5' -t 'zilla' -d
-```
-
-output:
-
-```text
-Client null sending CONNECT
-Client 2b77314a-163f-4f18-908c-2913645e4f56 received CONNACK (0)
-Client 2b77314a-163f-4f18-908c-2913645e4f56 sending SUBSCRIBE (Mid: 1, Topic: zilla, QoS: 0, Options: 0x00)
-Client 2b77314a-163f-4f18-908c-2913645e4f56 received SUBACK
-Subscribed (mid: 1): 0
-Client 2b77314a-163f-4f18-908c-2913645e4f56 received PUBLISH (d0, q0, r0, m0, 'zilla', ... (12 bytes))
-Hello, world
-```
-
-```bash
-mosquitto_sub -V '5' -t 'zilla' --cafile test-ca.crt -d
-```
-
-output:
-
-```text
-Client null sending CONNECT
-Client 26ab67d8-4a61-4e14-9d95-6a383c0cbdd7 received CONNACK (0)
-Client 26ab67d8-4a61-4e14-9d95-6a383c0cbdd7 sending SUBSCRIBE (Mid: 1, Topic: zilla, QoS: 0, Options: 0x00)
-Client 26ab67d8-4a61-4e14-9d95-6a383c0cbdd7 received SUBACK
-Subscribed (mid: 1): 0
-Client 26ab67d8-4a61-4e14-9d95-6a383c0cbdd7 received PUBLISH (d0, q0, r0, m0, 'zilla', ... (12 bytes))
-Hello, world
-```
-
-```bash
-mosquitto_pub -V '5' -t 'zilla' -m 'Hello, world' -d
-```
-
-output:
-
-```text
-Client null sending CONNECT
-Client 44181407-f1bc-4a6b-b94d-9f37d37ea395 received CONNACK (0)
-Client 44181407-f1bc-4a6b-b94d-9f37d37ea395 sending PUBLISH (d0, q0, r0, m1, 'zilla', ... (12 bytes))
-Client 44181407-f1bc-4a6b-b94d-9f37d37ea395 sending DISCONNECT
-```
-
-Check the internal mqtt-messages topic in Kafka
 ```bash
-kcat -C -b localhost:9092 -t mqtt-messages -J -u | jq .
-{
-  "topic": "mqtt-messages",
-  "partition": 0,
-  "offset": 0,
-  "tstype": "create",
-  "ts": 1683710037377,
-  "broker": 1,
-  "headers": [
-    "zilla:topic",
-    "zilla",
-    "zilla:local",
-    "44181407-f1bc-4a6b-b94d-9f37d37ea395",
-    "zilla:format",
-    "BINARY"
-  ],
-  "key": "zilla",
-  "payload": "Hello, world"
-}
+wget -qO- https://raw.githubusercontent.com/aklivity/zilla-examples/main/startup.sh | sh -s -- -m mqtt.kafka.broker
```

Comment:
Are we now directing folks to ignore the local checkout completely?
We seem to have lost a lot of context for those interacting GitHub-first.

Reply:
Feedback was discussed in a huddle. I have updated the mqtt.kafka.broker and quickstart readmes to be friendly to running locally.